src/components/modal/v2/parts/OfferCard.jsx

```diff
@@ -5,11 +5,13 @@ const OfferCard = ({ offer: { content, meta } }) => {
     const { termsLabel } = content;
     const aprRemoveTrailingZeros = meta?.apr.replace(/\D00$/, '');
     const aprFieldTitle = aprRemoveTrailingZeros === '0' ? termsLabel?.zeroApr : termsLabel?.nonZeroApr;
+    const offerHeaderField = termsLabel.offerHeader;
```

Comment:
```suggestion
    const offerHeaderField = termsLabel?.offerHeader;
```
to ensure termsLabel isnt coming in as undefined